specification/managednetworkfabric/resource-manager/readme.md

```diff
@@ -82,6 +82,16 @@ input-file:
   - Microsoft.ManagedNetworkFabric/preview/2023-02-01-preview/RoutePolicies.json
   - Microsoft.ManagedNetworkFabric/preview/2023-02-01-preview/common.json
 ```
+### Tag: package-2024-02-15-preview
+
+These settings apply only when `--tag=package-2024-02-15-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2024-02-15-preview'
+input-file:
+  - Microsoft.ManagedNetworkFabric/stable/2024-02-15-preview/InternetGatewayRules.json
```

Comment:
>stable [](http://example.com/codeflow?start=35&length=6)

This might be why the lintDiff and avocado checks fail - this should be preview and not stable. Hence the jsons cannot be found by tooling.